waiter/resources/web/error.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
```

Comment:
nitpick: can the `additional-info` section include the request method?

Reply:
The only valid method is GET.  I changed the handler to reflect that.